h/static/styles/annotation.scss

```diff
@@ -216,6 +215,17 @@ $annotation-card-left-padding: 10px;
   color: $text-color;
 }
 
+.share-dialog-msg {
+  color: $gray-light;
+  margin: -5px 10px 10px 10px;
+  line-height: 15px;
```

Comment:
Are these using typography elements from the new palette? If so, now would be a good time to introduce those in `variables.scss`.